### PR TITLE
feature/VDX-277

### PR DESCRIPTION
**Pull request by sksadjad**
- removed hardcoded templates
- fixed the energy shr portal
- fixed triall portal
- removed dbc related credential metadata from sphereon

### CODE REVIEW

File: packages/oid4vci-demo-frontend/src/configs/energy_shr.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "general": {
-    "credentialName": "Conference Attendee",
-    "issueCredentialType": "dbc2023",
+    "credentialName": "EnergySHR Guest",
+    "issueCredentialType": "energySHR",
     "oid4vpAgentBaseUrl": "https://ssi.dutchblockchaincoalition.org/demo/issuer",
```
Comment (nklomp) [inline packages/oid4vci-demo-frontend/src/configs/energy_shr.json]: Although this demo is currently not deployed, and although it is not part of this PR, please update all the ssi.dutchblockchaincoalition.org to ssi.energyshr.nl for the energyshr config(s)

**Review by nklomp — CHANGES_REQUESTED** [review]
Small remark about EnergySHR demo config

File: packages/oid4vci-demo-frontend/src/configs/energy_shr.json (hunk 1 above)
Comment (sksadjad) [reply packages/oid4vci-demo-frontend/src/configs/energy_shr.json]: changed to https://www.energyshr.nl/

**Review by nklomp — APPROVED** [review]
LGTM
